client/app/billing/autoRenew/ssh/user-ssh.html

```diff
@@ -39,14 +33,14 @@ <h1 data-translate="user_ssh_title"></h1>
             </button>
             <oui-dropdown-content>
                 <button class="oui-dropdown-option"
-                        data-ng-click="setAction('ssh/add/dedicated/user-ssh-add-dedicated', ctrlSsh.sshKeyList)"
+                        data-ng-click="setAction('ssh/add/dedicated/user-ssh-add-dedicated', ctrlSsh.sshKeyList, '', true)"
                         data-track-name="account::ssh::add::dedicated::add_ssh_key"
```

Comment:
We'll need to find a way to get rid of those `setAction` calls in favor of actual state transitions.